pkg/cmd/server/start/start_master.go

```diff
@@ -131,6 +123,20 @@ func NewCommandStartMaster(basename string, out io.Writer) (*cobra.Command, *Mas
 	return cmd, options
 }
 
+func MasterArgsAndFlags(flags *pflag.FlagSet) *MasterArgs {
```

Comment:
looks like these aren't used now... can we revert the change in this file and in start_node.go?